### DISCUSSION TITLE
Android - reanimated 2.14.1 - proguard rules

### DISCUSSION THREAD

**Discussion by GaelCO** [software-mansion/react-native-reanimated#3939] [Q&A] [answered]
Hi,

I updated reanimated from version 2.13.0 to 2.14.1.
Since then, the release version crashes with minifyEnabled set to true.
If I go back to version 2.13.0, the problem disappears.

I didn't see any updates on proguard rules in the documentation.
Should rules be added for certain packages or dependencies ?

NB : my actual proguard rules for reanimated :
-keep class com.swmansion.reanimated.** { *; }
-keep class com.facebook.react.turbomodule.** { *; }

**Comment by GaelCO** [comment]
I think my issue is same as https://github.com/software-mansion/react-native-reanimated/issues/3952

**Comment by tomekzaw** [comment]
Hey @GaeICO, thanks for reporting this problem. I'm not aware of any changes between 2.13.0 and 2.14.1 that would affect Proguard rules.
1. Can you share with us the output of logcat?
2. Can you check if #3494 resolves the issue?

**Reply by GaelCO** [reply]
Thanks for your reply

My issue is not related to proguard. If I disable it, my app still crash.
But, If I apply the workaround proposed by @mrbrentkelly : https://github.com/software-mansion/react-native-reanimated/issues/3952#issuecomment-1384566417, the issue disappears.

So it's the same problem as https://github.com/software-mansion/react-native-reanimated/issues/3952

**Comment by GaelCO** [comment, marked as answer]
This issue is fixed by version 2.14.2 :
Disable HERMES_ENABLE_DEBUGGER when building Reanimated 2 AAR package by @tomekzaw in https://github.com/software-mansion/react-native-reanimated/pull/3956
